docs/specs/backend_internal.yaml

```diff
@@ -0,0 +1,939 @@
+openapi: 3.0.1
+info:
+  title: 3scale Backend Internal API
+  version: 1.0.0
+servers:
+- url: http://localhost:3000/internal
```

Comment:
```suggestion
- url: http://localhost:3001/internal
```

Because normally backend runs on `3001`, while `3000` is used for porta

Reply:
But we can leave it until further spec improvements.

Reply:
I pushed the change